tee-worker/omni-executor/pumpx/Cargo.toml

```diff
@@ -5,6 +5,13 @@ authors = ['Trust Computing GmbH <info@litentry.com>']
 edition.workspace = true
 
 [dependencies]
+hex = { workspace = true }
+jsonrpsee = { workspace = true, features = ["client"] }
+serde = { version = "1.0", features = ["derive"] }
+serde_json = { workspace = true }
+serde_with = { version = "3.11", features = ["hex"] }
```

Comment:
Same here - maybe define it in workspace? I mean it would make sure we use consistent version in the whole project